1.8/tutorial/loadbalancing.md

```diff
@@ -0,0 +1,41 @@
+
+---
+post_title: DC/OS 101 - Loadbalancing
+menu_order: 3
+---
+
+# Prerequisites
+We expect you to have access to a running DC/OS OSS cluster and the DC/OS CLI installed and configured.
+Furthermore, you have just deployed app2 and marathon-lb in our cluster and have verified both are running.
+
+# Objective
+In the last session, we have learned how to scale our application.
+But if there multiple instances of an app in my cluster, how do cluster internal services or external request decide which instance should be used?
+
+# Steps
+  This is the topic of load balancing. With DC/OS you have two different inbuilt options: [Marathon-LB](https://dcos.io/docs/1.8/usage/service-discovery/marathon-lb/) and [Named virtual IPs](https://dcos.io/docs/1.8/usage/service-discovery/load-balancing-vips/).
+  We have seen both before in the context of service discovery and especially marathon-lb when making app2 publically available.
+  Now we will explore both options in more detail.
+  * Therefore we first scale app2 to two instances: `dcos marathon app update /dcos-101/app2 instances=2`
+  * Marathon-lb
+      * Check the app2 as before via http://<public-node>:10000, when doing this repeatedly you should see the request being served by different instances.
```

Comment:
+      * Check the app2 as before via http://<public-node>:10000. When you do this repeatedly you should see the request being served by different instances of app2.